### ISSUE TITLE
Scale of parent object is not applied to dynamic collider

### ISSUE THREAD

**Issue by flo-bit** [threlte/threlte#803]
Thanks a lot for this great project, really like threlte so far!

When adding a collider set to "dynamic" to a T.Mesh as a child and updating the meshes position or rotation, the position and rotation of the collider also updates correctly. That doesn't work with scaling.

See a slightly modified version that shows the issue of the ["Your first scene" project here.](https://stackblitz.com/edit/udnfbu?file=src%2Fexample%2FScene.svelte). As you can see rotation and position update, but when hovering over the cuboid the scale of the collider (yellow outlines) doesn't update.

![scale](https://github.com/threlte/threlte/assets/45694132/52da915c-ce40-416a-940d-eae93cac64e0)

**Comment by alexco95** [comment]
Hi,

I have the same issue. Any updates on this?

By the way, I'm loving Threlte so far! Just started learning it this week.

**Comment by DefinitelyMaybe** [comment]
oh man. scratching my head on this one. So in [source](https://github.com/threlte/threlte/blob/6ebc7cffdef5bf0e10201e26d15850cace13cd2f/packages/rapier/src/lib/components/Colliders/Collider.svelte#L151) the colliders translation and rotation are set but not it's scale... not sure why thats was left out. Lemme ask the team.

**Comment by flo-bit** [comment]
Yeah I also took a look at the source when I opened this issue, but there didn't seem to be a easy fix, how I got around the problem in my project was by just recreating the component each time the scale changes for example by wrapping it into `{#key scale} component {/key}` tags. Of course this has performance drawbacks if you change the scale continuously, but works well enough if you just change it once.

**Comment by DefinitelyMaybe** [comment]
wow! someone on discord found another solution. Thank you rubber-duck + types.

The [refresh function](https://github.com/threlte/threlte/blob/6ebc7cffdef5bf0e10201e26d15850cace13cd2f/packages/rapier/src/lib/components/Colliders/AutoColliders.svelte#L120) from the component can be manually called like so:

```svelte
<AutoColliders shape={"cuboid"} let:refresh>
    <Text3DGeometry on:create={refresh} />
</AutoColliders>
```

or use `bind` instead of `let` if needed within the script.
